src/init.ts

```diff
@@ -49,12 +50,24 @@ export async function init(signerService: SignerService) {
     const messagingService = await MessagingService.create(signerService);
 
     async function connectToCacheServer() {
+        const chainId = signerService.chainId;
+        const stakingPoolFactoryAddress = chainConfigs()[chainId].stakingPoolFactoryAddress;
+
         const cacheClient = new CacheClient(signerService);
         await cacheClient.init();
         await cacheClient.login();
         const domainsService = await DomainsService.create(signerService, cacheClient);
-        const stakingService = await StakingService.create(signerService, domainsService);
-        const stakingPoolService = await StakingFactoryService.create(signerService, domainsService);
+
+        const stakingAddressProvided = stakingPoolFactoryAddress && stakingPoolFactoryAddress.length;
```

Comment:
This is a test for the stakingPoolFactoryAddress
```suggestion
        const stakingPoolFactoryAddressProvided = stakingPoolFactoryAddress && stakingPoolFactoryAddress.length;
```